helper-cli/src/main/kotlin/commands/ListStoredScanResultsCommand.kt

```diff
@@ -54,18 +55,17 @@ internal class ListStoredScanResultsCommand : CliktCommand(
 
     private val configArguments by option(
         "-P",
-        help = "The comma separated list of key-value pairs to override in the configuration file. For example: " +
+        help = "Override a key-value pair in the configuration file. For example: " +
                 "-P scanner.postgresStorage.schema=testSchema"
-    ).convert { csv ->
-        csv.split(",").map { keyValuePair ->
-            keyValuePair.split("=").let { (k, v) ->
-                Pair(k, v)
-            }
-        }
-    }.default(emptyList())
+    ).convert { Pair(it.substringBefore("="), it.substringAfter("=", "")) }
+        .multiple()
+        .unique()
 
     override fun run() {
         val config = OrtConfiguration.load(mapOf(*configArguments.toTypedArray()), configFile)
+        val x = mapOf(*configArguments.toTypedArray())
```

Comment:
Please remove this left-over debug statement.

Reply:
Done.